x-pack/plugin/sql/qa/mixed-node/src/test/resources/all_field_types.json

```diff
@@ -0,0 +1,59 @@
+      "properties": {
```

Comment:
minor: why are these entries indented like this? (many whitespaces from the beginning of the line)